alphastats/statistics/DifferentialExpressionAnalysis.py

```diff
@@ -15,34 +16,34 @@ def __init__(
         sample: str,
         index_column: str,
         preprocessing_info: Dict,
-        # TODO move these to perform()?
         group1: Union[str, list],
         group2: Union[str, list],
         column: str = None,
+        # TODO move these to perform()?
         method: str = "ttest",
         perm: int = 10,
         fdr: float = 0.05,
     ):
         self.mat = mat
-        self.metadata = metadata
+
         self.sample = sample
         self.index_column = index_column
         self.preprocessing_info = preprocessing_info
 
-        self.group1 = group1
-        self.group2 = group2
-        self.column = column
```

Comment:
As far as I can see this should cause problems on line 37:
if isinstance(self.group1 ...

Reply:
Went by commits. Was resolved later